packages/wdio-types/src/Capabilities.ts

```diff
@@ -558,6 +558,139 @@ export interface AppiumIOSCapabilities {
     webkitDebugProxyPort?: number;
 }
 
+/**
+ * Appium xcuitest Capabilities
+ *
+ * @see https://github.com/appium/appium-xcuitest-driver
+ */
+export interface AppiumXCUITestCapabilities {
+    'appium:platformName'?:                         string;
```

Comment:
Can we remove the spaces and remove the `;` at the end, just so we stay compliant with the rest of the code style, e.g.:
```suggestion
    'appium:platformName'?: string
```

Reply:
Sure Christian will do that 🎉